### PR TITLE
DAOS-9321 doc: Updates to QSG for 2.0

### PR DESCRIPTION
**Pull request by ryon-jensen**
Fixed several commands in listed examples to refect
command updates. Updated references from 1.2 to 2.0.

Signed-off-by: Ryon Jensen <ryon.jensen@intel.com>

Doc-only: true

### CODE REVIEW

**Comment by johannlombardi** [comment]
Merge master to address conflict with a patch i merged.

File: docs/QSG/tour.md [hunk 1]
```diff
@@ -242,11 +242,13 @@ bring-up DAOS servers and clients.
 	$ dmg pool list
 	Pool UUID								Svc Replicas
 	---------								------------
-	5f362dc2-6154-44c7-8348-9de6f0a3d5d1	0
+	bf9a1b61-e3ba-4dca-a4ba-f35776edc822	0
+	13aaa333-1533-41cb-85d8-8a22728cfbad	0
+	f73eefa5-937d-4f0d-8934-4b3192dba4e4	0
```
Comment (johannlombardi) [inline docs/QSG/tour.md:247]: The pool list output is significantly different on 2.0. This needs to be updated.

File: docs/QSG/tour.md [hunk 2]
```diff
@@ -376,7 +378,7 @@ bring-up DAOS servers and clients.
 	0d121c02-a42d-4029-8dce-3919b964b7b3
 
 ### daos container destroy
-
+    $ export export DAOS_CONT=bc4fe707-7470-4b7d-83bf-face75cc98fc
```
Comment (johannlombardi) [inline docs/QSG/tour.md:381]: we should use container label instead of UUID.

**Review by johannlombardi — CHANGES_REQUESTED** [review]
This should be changed to use container labels everywhere IMO (no UUID).

Shouldn't docs/QSG/setup_suse.md be modified too?

**Comment by ryon-jensen** [comment]
> Merge master to address conflict with a patch i merged.

@johannlombardi - This is for 2.0 so I'm not sure we want to merge master into it?

File: docs/QSG/tour.md (hunk 1 above)
Comment (ryon-jensen) [reply docs/QSG/tour.md:247]: Will update, thanks.

File: docs/QSG/tour.md (hunk 2 above)
Comment (ryon-jensen) [reply docs/QSG/tour.md:381]: I'll change everywhere to use cont label.

**Review by cdurf1 — DISMISSED** [review]

**Comment by ryon-jensen** [comment]
> Shouldn't docs/QSG/setup_suse.md be modified too?

@johannlombardi - yes, probably. There's another ticket (DAOS-8574) for running through the quick start guide on SLES that Kal is working on. If he identifies gaps I can include them in this PR.

**Comment by johannlombardi** [comment]
Sorry for the confusion, i meant that i merged release/2.0 since i landed a patch conflicting with this PR. That being said, this PR will eventually have to ported to master too to keep it in sync.
As for the suse quickstart, i would be proactive and make sure that any changes we make to the CentOS one are also included to the suse one.

**Review by cdurf1 — APPROVED** [review]

**Review by cdurf1 — DISMISSED** [review]

File: docs/QSG/tour.md [hunk 3]
```diff
@@ -219,13 +218,13 @@ bring-up DAOS servers and clients.
 	A::EVERYONE@:
 
 	# Update pool access entry for the existing test_group1 to no-permission
-	dmg pool update-acl -e A:G:test_group1@: --pool=$DAOS_POOL
```
Comment (ravalsam) [inline docs/QSG/tour.md:222]: --pool is not available so can be removed
`dmg pool update-acl --help
Usage:
  dmg [OPTIONS] pool update-acl [update-acl-OPTIONS] [<pool name or UUID>]

Application Options:
      --allow-proxy              Allow proxy configuration via environment
  -l, --host-list=               A comma separated list of addresses <ipv4addr/hostname> to connect to
  -i, --insecure                 Have dmg attempt to connect without certificates
  -d, --debug                    Enable debug output
  -j, --json                     Enable JSON output
  -J, --json-logging             Enable JSON-formatted log output
  -o, --config-path=             Client config file path

Help Options:
  -h, --help                     Show this help message

[update-acl command options]
      -a, --acl-file=            Path for new Access Control List file
      -e, --entry=               Single Access Control Entry to add or update`

File: docs/QSG/tour.md [hunk 4]
```diff
@@ -219,13 +218,13 @@ bring-up DAOS servers and clients.
...
+	dmg pool update-acl -e A:G:test_group1@: --pool=Pool3
 
 	# Update pool access entry for a new user test_user3 with rw permission
-	dmg pool update-acl -e A::test_user3@:rw --pool=$DAOS_POOL
```
Comment (ravalsam) [inline docs/QSG/tour.md:225]: --pool can be removed

File: docs/QSG/tour.md [hunk 5]
```diff
@@ -219,13 +218,13 @@ bring-up DAOS servers and clients.
...
+	dmg pool update-acl -e A::test_user3@:rw --pool=Pool3
 
 	# Get pool security acl after update-acl
-	$ dmg pool get-acl --pool=$DAOS_POOL
```
Comment (ravalsam) [inline docs/QSG/tour.md:228]: --pool can be removed

File: docs/QSG/tour.md [hunk 6]
```diff
@@ -308,7 +313,7 @@ bring-up DAOS servers and clients.
 
 	use 'daos help RESOURCE' for resource specifics
 
-	$ daos help cont
+	$ daos cont --help
 	daos command (v1.2), libdaos 1.2.0
```
Comment (ravalsam) [inline docs/QSG/tour.md:317]: There is no version info so maybe remove this line and update the help?

` daos cont --help
Usage:
  daos [OPTIONS] container <command>

Application Options:
      --debug     enable debug output
      --verbose   enable verbose output (when applicable)
  -j, --json      enable JSON output

Help Options:
  -h, --help      Show this help message

Available commands:
  check          check objects' consistency in a container
  clone          clone a container
  create         create a container
  create-snap    create container snapshot (aliases: snap)
  del-attr       delete container user-defined attribute (aliases: delattr)
  delete-acl     delete a container's ACL
  destroy        destroy a container
  destroy-snap   destroy container snapshot
  get-acl        get a container's ACL
  get-attr       get container user-defined attribute (aliases: getattr)
  get-prop       get container user-defined attribute (aliases: getprop)
  list           list all containers in pool (aliases: ls)
  list-attr      list container user-defined attributes (aliases: list-attrs, lsattr)
  list-objects   list all objects in container (aliases: list-obj)
  list-snap      list container snapshots (aliases: list-snaps)
  overwrite-acl  replace a container's ACL (aliases: replace)
  query          query a container
  set-attr       set container user-defined attribute (aliases: setattr)
  set-owner      change ownership for a container (aliases: chown)
  set-prop       set container user-defined attribute (aliases: setprop)
  stat           get container statistics
  update-acl     update a container's ACL
`

File: docs/QSG/tour.md [hunk 7]
```diff
@@ -525,7 +540,7 @@ bring-up DAOS servers and clients.
 	# dmg pool destroy Timeout or failed due to pool has active container(s)
 	# Workaround pool destroy --force option
 
-		$ dmg pool destroy --pool=$DAOS_POOL --force
+		$ dmg pool destroy --pool=Pool1 --force
```
Comment (ravalsam) [inline docs/QSG/tour.md:543]: --pool can be removed

**Review by ravalsam — COMMENTED** [review]
Overall looks good, There are a few comments from my side. Once it's updated we are good to merge.

**Comment by ryon-jensen** [comment]
> Overall looks good, There are a few comments from my side. Once it's updated we are good to merge.

Made updates based on your feedback @ravalsam. Kal says he's good with it from the SLES side.

**Review by ravalsam — APPROVED** [review]

**Review by ravalsam — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Comment by johannlombardi** [comment]
Please port to master.

**Pull request merged**